### ISSUE TITLE
Duration and distance for waypoints.

### ISSUE THREAD

**Issue by prathammehta** [bramus/react-native-maps-directions#46]
The waypoint feature is awesome! Can you also expose the Duration and distance to each waypoint in the callback for onReady? 
I believe Google's direction API response already contains that information.

Thanks for a great library!

**Comment by why-absystem** [comment]
@prathammehta , u can also get duration and distance from result. 
result.duration
result.distance

**Comment by prathammehta** [comment]
Yup. I know about that. It'll be awesome to be able to get the same information for each waypoint as well.

**Comment by rajeshsmallarc** [comment]
hi @prathammehta 

I am getting an unknown warning when using waypoints.

Can you please provide an example.

**Comment by bramus** [comment]
Passing the original Google Directions response into the `onReady` callback looks like a good solution, to cater for more advanced/specific use cases.

I'll happily accept a PR that implements this.

**Comment by garydevenay** [comment]
I have created a pull request which includes a fix for this issue. https://github.com/bramus/react-native-maps-directions/pull/85

This includes each step in a journey, each which has its own individual duration.
